### DISCUSSION TITLE
Trying to preview web service on proxy mode, but nothing is showing up on the page.

### DISCUSSION THREAD

**Discussion by jasongatech** [coder/code-server#5100] [Q&A]
I have a nextJS project with all the requirements installed already, but when I use proxy mode with subpath (/proxy/port) nothing shows up on the page. When I open up the developer mode, it shows all the elements just fine, but the proxy mode shows nothing (basically just a white blank page). 

Not really sure why this happens, and not sure if it is problem with proxy mode or just that there was something wrong with my npm set up.

Anyone has had the same problem?

Below is the screenshot of the console showing up the error... Not really sure how to fix them :/
![캡처](https://user-images.githubusercontent.com/85703335/163342919-60541656-5481-4ee3-a2b1-f381b76d6e8b.PNG)

**Comment by jsjoeio** [comment]
With CRA, you have to do some extra config stuff. It may be similar with Next. Have you tried [this](https://coder.com/docs/code-server/latest/guide#proxying-to-create-a-react-app)?
